qa/workunits/cephadm/test_cephadm.sh

```diff
@@ -313,7 +313,8 @@ $CEPHADM enter --fsid $FSID --name mgr.x -- test -d /var/lib/ceph/mgr/ceph-x
 $CEPHADM enter --fsid $FSID --name mon.a -- pidof ceph-mon
 expect_false $CEPHADM enter --fsid $FSID --name mgr.x -- pidof ceph-mon
 $CEPHADM enter --fsid $FSID --name mgr.x -- pidof ceph-mgr
-expect_false $CEPHADM --timeout 1 enter --fsid $FSID --name mon.a -- sleep 10
+# this triggers a bug in older versions of podman, including 18.04's 1.6.2
```

Comment:
The ref'd libpod issue claims fixed in 1.5.1, but we see this on 1.6.2? 

Might be nice if we could still run the tests when `podman_ver >= fixed_ver`

Reply:
Yeah, :man_shrugging: 1.6.2-1~ubuntu18.04~ppa1 is the ubunut package version, and it was definitely happening on ubuntu podman only, and commenting out that line fixed it.  Maybe it was a different/related bug that followed.